distro/Makefile

```diff
@@ -29,13 +30,23 @@ get-artifacts:
 	# get artifacts
 	cd .. && mvn clean package && cd -
 	mkdir -p ${DIST}
-	wget -O ${DIST}/${HZ_INSTALL_TAR} "https://download.hazelcast.com/download.jsp?version=hazelcast-${HZ_VERSION}&type=tar&p="
+	mvn dependency:get \
+			-DrepoUrl=https://oss.sonatype.org/content/repositories/snapshots/ \
```

Comment:
This url is correct for snapshot, this script will run also for release version, we need to check the HZ_VERSION and if it doesn't contain `-SNAPSHOT` it should use `https://repo1.maven.org/maven2/`